include/mbedtls/config.h

```diff
@@ -2567,6 +2578,28 @@
  */
 #define MBEDTLS_XTEA_C
 
+/**
+ * \def MBEDTLS_SERIALIZE_C
+ *
+ * Enable the serialization module. This is needed for offloading (see
+ * MBEDTLS_FS_IO_ALT).
```

Comment:
Not knowing what offloading means, I expected an explanation at `MBEDTLS_FS_IO_ALT`, but there is none. Could you elaborate somewhere?